webapp/src/main/java/com/box/l10n/mojito/quartz/QuartzConfig.java

```diff
@@ -25,37 +27,45 @@ public class QuartzConfig {
 
   public static final String DYNAMIC_GROUP_NAME = "DYNAMIC";
 
-  @Autowired Scheduler scheduler;
+  @Autowired QuartzSchedulerManager schedulerManager;
 
   @Autowired(required = false)
   List<Trigger> triggers = new ArrayList<>();
 
   @Autowired(required = false)
   List<JobDetail> jobDetails = new ArrayList<>();
 
-  @Autowired QuartzPropertiesConfig quartzPropertiesConfig;
+  @Value("${l10n.org.quartz.scheduler.enabled:true}")
+  Boolean schedulerEnabled;
 
   /**
    * Starts the scheduler after having removed outdated trigger/jobs
    *
    * @throws SchedulerException
    */
   @PostConstruct
-  void startScheduler() throws SchedulerException {
-    Properties quartzProps = quartzPropertiesConfig.getQuartzProperties();
+  void startSchedulers() throws SchedulerException {
     removeOutdatedJobs();
-    if (Boolean.parseBoolean(quartzProps.getProperty("org.quartz.scheduler.enabled", "true"))) {
-      logger.info("Starting scheduler");
-      scheduler.startDelayed(2);
+    int delay = 2;
+    if (schedulerEnabled) {
+      for (Scheduler scheduler : schedulerManager.getSchedulers()) {
+        logger.info("Starting scheduler: {}", scheduler.getSchedulerName());
+        scheduler.startDelayed(delay);
+        delay++;
```

Comment:
I'm incrementing the delay for each scheduler as without it on startup lock errors were observed from schedulers starting at the same time. The errors don't cause a startup failure (the scheduler with the error just restarts and comes up successfully) but staggering the startups like this helps to avoid the errors and reduce logging noise on application startup.

Reply:
discussed offline but for the record, this is worth adding as comment in the code so we remember the reasoning behind